src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,9 @@ Release History
 ===============
 upcoming
 ++++++
+
+0.3.55
+++++++
 * 'az containerapp create': Support --registry-identity to use Environment identity for ACR authentication.
```

Comment:
only support for create? what about update?

Reply:
For `az containerapp update`, we don't support registry update. Customer need to use `az containerapp registry set`